docs/notebooks/util/plotting.py

```diff
@@ -16,10 +16,11 @@
 import numpy as np
 from matplotlib import cm
 
+from trieste.type import TensorType
 from trieste.utils import to_numpy
 
 
-def create_grid(mins, maxs, grid_density=20):
+def create_grid(mins: TensorType, maxs: TensorType, grid_density=20):
```

Comment:
if this can only be a 2 element TensorType, then maybe you should assert that below.

Reply:
there is a lot of stuff in these files that needs fixing, a lot. I have another PR which completely refactors them, so I don't think it's worth adding things here. Sound reasonable?

Reply:
i don't suggest reviewing these files - they're not in a good way and I've completely refactored them elsewhere. I'll make the changes you've suggested

Reply:
Can't really say without seeing the other PR, but in general I don't support the argument that because there's lots of stuff that needs fixing you shouldn't fix things reviewers point out. 

That said all my comments are non-critical is why I approved the PR (so you can merge if you want).

Reply:
yeah i think i was just being lazy, you got me

Reply:
haha, no worries. As I said, optional comment, but thanks for making the changes.